Sources/OnboardingFeature/OnboardingView.swift

```diff
@@ -552,6 +550,8 @@ private let onboardingGameReducer = gameReducer(
   isHapticsEnabled: { _ in true }
 )
 
+private struct DelayedNextStepId: Hashable {}
```

Comment:
Why'd this move?

Reply:
I had it at the bottom of the reducer and Swift complains it will never be excuted lol. It’s a Swift bug I should file but moved it out of the reducer for our episodes next week 🤫